packages/teams-js/src/public/sharing.ts

```diff
@@ -10,7 +10,12 @@ import { runtime } from './runtime';
  * For more info, see [Share to Teams from personal app or tab](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab)
  */
 export namespace sharing {
+  /** shareWebContent callback function type */
+  export type shareWebContentCallbackFunctionType = (err?: SdkError) => void;
```

Comment:
>export [](http://example.com/codeflow?start=2&length=6)

Should this really be exported and documented? It's only ever used once internally in this file.